nodes/widgets/locales/en-US/ui_number_input.html

```diff
@@ -0,0 +1,28 @@
+<script type="text/html" data-help-name="ui-number-input">
+    <p>
+        Adds a single number input row to your dashboard
+    </p>
+    <h3>Dynamic Properties (Inputs)</h3>
+    <p>Any of the following can be appended to a <code>msg.</code> in order to override or set properties on this node at runtime.</p>
+    <dl class="message-properties">
+        <dt class="optional">class <span class="property-type">string</span></dt>
+        <dd>Add a CSS class, or more, to the Button at runtime.</dd>
+        <dt>Range <span class="property-type">number</span></dt>
+        <dd>
+            min - the minimum value the slider can be changed to; max - the maximum value the
+            slider can be changed to; step - the increment/decrement value when the slider is moved.
+        </dd>
+    </dl>
```

Comment:
any reason why the `Range` is _inside_ this `<dl>`? It would make more sense if min/step/max where all int he same `<dl>` instead

Reply:
I took the reference from ui-slider.html when I was writing the above. fixed it
https://github.com/FlowFuse/node-red-dashboard/blob/511f310947d956a080526bad9acaff738b774025/nodes/widgets/locales/en-US/ui_slider.html#L18